torch/utils/mkldnn.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+
+
+def to_mkldnn(module):
```

Comment:
can you just call _apply?

Reply:
`_apply` doesn't pass the module itself to the callback, but we need it to check whether it's conv (and if yes we need to do extra conversion).

Reply:
If you just want to match children modules - there's .apply